url.bs

```diff
@@ -272,6 +272,97 @@ for further processing.
 U+0020 SPACE, U+0023 (#), U+0025 (%), U+002F (/), U+003A (:), U+003F (?), U+0040 (@), U+005B ([),
 U+005C (\), or U+005D (]).
 
+<p>A <a for=/>host</a>'s <dfn for=host export id=concept-host-public-suffix>public suffix</dfn> is
+the portion of a <a for=/>host</a> which is controlled by a registrar, public or otherwise. To
+obtain <var>host</var>'s <a for=host>public suffix</a>, run the following steps:
+
+<ol>
+ <li><p>Let <var>parsed</var> be the result of <a lt="host parser">host parsing</a> <var>host</var>.
```

Comment:
A host is already parsed (otherwise it wouldn't be a host). You also need to introduce the _host_ variable in the paragraph before the algorithm.

Reply:
> A host is already parsed (otherwise it wouldn't be a host).

Hrm. Yeah, I guess it's reasonable to assume that we'll only be using this algorithm on already-parsed hosts.

> You also need to introduce the host variable in the paragraph before the algorithm.

Line 277 introduces `<var>host</var>`. Would you prefer to be more explicit, like "`To obtain the <a for=host>public suffix</a> for a <a for=/>host</a> <var>A</var>:`"?

Reply:
Ah, I missed that on line 277. That's fine, but your alternative here works too.